services/resource-coordinator/src/drivers/kubernetes/KubernetesDriver.js

```diff
@@ -0,0 +1,203 @@
+const { BaseDriver } = require('@openintegrationhub/resource-coordinator');
```

Comment:
I think that kubernetes driver is common solution enough to put it into src/resource-coordinator along with base-driver.
Anyway (at least at this moment) we do not plan to have other backends except k8s.